File: Entidades/NVRemitoEN.vb

Public Class NVRemitoEN
    Private _codremito As Integer
    Public Property CodRemito() As Integer
        Get
            Return _codremito
        End Get
        Set(ByVal value As Integer)
            _codremito = value
        End Set
    End Property

    Private _codnot As Integer
    Public Property CodNot() As Integer
        Get
            Return _codnot
        End Get
        Set(ByVal value As Integer)
            _codnot = value
        End Set
    End Property

    Private _nroremito As Integer
    Public Property NroRemito() As Integer
        Get
            Return _nroremito
        End Get
        Set(ByVal value As Integer)
            _nroremito = value
        End Set
    End Property

    Private _fecha As DateTime
    Public Property Fecha() As DateTime
        Get
            Return _fecha
        End Get
        Set(ByVal value As DateTime)
            _fecha = value
        End Set
    End Property


    Private _detalle As List(Of DetalleEN)
    Public Property Detalle() As List(Of DetalleEN)
        Get
            Return _detalle
        End Get
        Set(ByVal value As List(Of DetalleEN))
            _detalle = value
        End Set
    End Property
End Class

File: Entidades/NVRemitoEN.cs
using System;
using System.Collections.Generic;

namespace Entidades
{
    public class NVRemitoEN
    {
        private int _codremito;

        public int CodRemito
        {
            get
            {
                return _codremito;
            }

            set
            {
                _codremito = value;
            }
        }

        private int _codnot;

        public int CodNot
        {
            get
            {
                return _codnot;
            }

            set
            {
                _codnot = value;
            }
        }

        private int _nroremito;

        public int NroRemito
        {
            get
            {
                return _nroremito;
            }

            set
            {
                _nroremito = value;
            }
        }

        private DateTime _fecha;

        public DateTime Fecha
        {
            get
            {
                return _fecha;
            }

            set
            {
                _fecha = value;
            }
        }

        private List<DetalleEN> _detalle;

        public List<DetalleEN> Detalle
        {
            get
            {
                return _detalle;
            }

            set
            {
                _detalle = value;
            }
        }
    }
}